pkg/config/config.go

```diff
@@ -35,6 +35,8 @@ type AdditionalDirectory struct {
 type Finch struct {
 	CPUs   *int    `yaml:"cpus"`
 	Memory *string `yaml:"memory"`
+	//Soci: boolean value of whether user wants SOCI installed
+	Soci *bool `yaml:"soci_snapshotter,omitempty"`
```

Comment:
what do you think about making this `Snapshotter` or `ContainerdSnapshotter` ? This could leave the door open for adding support for more snapshotters in the future